crates/ol/stf/src/transaction_processing.rs

```diff
@@ -175,7 +192,13 @@ fn apply_interactions<S: IStateAccessor>(
     fx_buf: AcctInteractionBuffer,
     context: &BasicExecContext<'_>,
 ) -> ExecResult<()> {
-    // Send the messages off to each of the targets.
+    // Process transfers: pure value transfers with no message data
+    for (dest, amount) in fx_buf.transfers {
+        let payload = MsgPayload::new(amount, vec![]);
+        account_processing::process_message(state, source, dest, payload, context)?;
```

Comment:
what does `process_message` do? It seems to me that this should only be "send message", where we insert a transfer message into the destination inbox, but nothing more than that.